test/integration/api.bats

```diff
@@ -33,9 +33,31 @@ function teardown() {
 	skip
 }
 
-# FIXME
 @test "docker cp" {
```

Comment:
Actually, would it be possible to check the content of what we copy rather than just if the file exists?

What about something like this:

```bash
test_file="/bin/busybox"
temp_dest=`mktemp -d` # create a temporary destination directory

# create the container
run docker_swarm run -d --name test_container busybox sleep 500
[ "$status" -eq 0 ]

# grab the checksum of the test file inside the container.
run docker_swarm exec test_container md5sum /bin/busybox
[ "$status" -eq 0 ]
container_checksum=$output

# copy the test file from the container to the host.
run docker_swarm cp $container_id:$test_file $temp_dest
[ "$status" -eq 0 ]
[ -f $temp_dest/`basename $test_file` ]

# compute the checksum of the copied file.
run md5sum $temp_dest/`dirname $test_file`
[ "$status" -eq 0 ]
host_checksum=$output

# Verify that they match.
[ $container_checksum == $host_checksum ]
```